src/applications/my-education-benefits/containers/IntroductionPage.jsx

```diff
@@ -28,29 +24,6 @@ export const IntroductionPage = ({ user, route }) => {
     />
   );
 
-  const handleRedirect = async response => {
```

Comment:
The introduction page was making its own call to check if it needed to redirect. This also created a duplicate call. Since the form system already makes a call on its own, I moved redirect logic to the actions.